bot/app/database/db_objects.py

```diff
@@ -0,0 +1,214 @@
+                (self.e_name, self.date, self.time, self.admission, self.desc, \
+                self.e_pic_id))
+            self.conn.commit()
+        except sqlite3.IntegrityError as e:
+            logging.error(e)
+        
+        curs.close()
+        conn.close()
+        
+class Artist:
+
+    def __init__(self, a_name, webs, soundc, bandc, bio, a_pic_id):
+
+        self.a_name = a_name
+        self.webs = webs
+        self.soundc = soundc
+        self.bandc = bandc
+        self.bio = bio
+        self.a_pic_id = a_pic_id
+
+        self.insert_artist()
+        
+    def insert_artist(self):
+        
+        conn = sqlite3.connect(DB_NAME)
+        curs = conn.cursor()
+
+        try:
+            curs.execute("""
+                insert into Artists values (?,?,?,?,?,?)""",
+                (self.a_name, self.webs, self.soundc, self.bandc, self.bio, \
+                self.a_pic_id))
+            self.conn.commit() 
+        except sqlite3.IntegrityError as e:
+            logging.error(e)
+        
+        curs.close()
+        conn.close()
+
+class PlaysAt:
+
+    """
+    Instance represents Artist-Event connection (aka playsAt Table)
+    """
+
+    def __init__(self):
+        
+        self.a_name = input("Enter the Artist's name: ")
+        self.e_name = input("Enter the Event's name: ")
+        self.date = input("Enter the Event's date (format DD.MM.YYYY): ")
+
+        self.check_for_artist()
+        self.check_for_event()
+        self.insert_plays_at()
+
+    def check_for_artist(self):
+        
+        conn = sqlite3.connect(DB_NAME)
+        curs = conn.cursor()
+        
+        curs.execute("select * from Artists where a_name=?", (self.a_name, ))
+        artist = curs.fetchone()
+        
+        if artist:
+            print("Corresponding artist found! Check: \n", artist)
+            answ = input("Is this the right entry? [Y/n]" )
+            if answ == "Y":
+                pass
+            else:
+                self.a_name = input("Please re-enter artist-name: ")
+                check_for_artist()
+        else: 
+            self.a_name = input("Please re-enter artist-name: ")
+            self.check_for_artist()
+
+        curs.close()
+        conn.close()
+
+    def check_for_event(self):
+        
+        conn = sqlite3.connect(DB_NAME)
+        curs = conn.cursor()
+        
+        curs.execute("select * from Events where e_name=? and date=?", (self.e_name, self.date))
+        event = curs.fetchone()
+
+        if event:
+            print("Corresponding event found! Check: \n", event)
+            answ = input("Is this the right entry? [Y/n]" )
+            if answ == "Y":
+                pass
+            else:
+                self.e_name = input("Please re-enter event-name: ")
+                check_for_event()
+        else: 
+            self.e_name = input("Please re-enter event-name: ")
+            self.check_for_event()
```

Comment:
Same as above

Reply:
will stay print for now as quick`n dirty solution to output on `stdout` and not log to file

Reply:
[This](https://github.com/wuest-halle/wuestbot/pull/2#discussion_r341818515) is the comment I'm refering to, for the recursive call with user input.

Reply:
fixed